_posts/2023-06-16-deploy-on-kubernetes-with-helm.adoc

```diff
@@ -0,0 +1,400 @@
+[INFO] #9 naming to docker.io/library/helloworld:latest done
+[INFO] #9 DONE 0.8s
+[INFO] Successfully built application image helloworld:latest
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time:  34.784 s
+[INFO] Finished at: 2023-06-14T17:27:40+02:00
+[INFO] ------------------------------------------------------------------------
+
+----
+And then make sure you have it in your docker images
+[source,nowrap]
+----
+ $ docker images
+REPOSITORY                        TAG                     IMAGE ID       CREATED          SIZE
+helloworld                        latest                  6d404b584da2   32 seconds ago   686MB
+
+----
+
+## Push the just created image to a container registry (e.g. quay.io, ghcr.io or hub.docker.com)
+
+In this article we will use Quay.io as a container registry.
+If you don't have an account you can start a free trial on https://quay.io/[https://quay.io/] or create an account on https://hub.docker.com/[dockerhub].
+
+* Login to your container registry (see how to get started https://docs.quay.io/solution/getting-started.html[here])
+
+* Tag the new image with the proper format 'quay.io/username/reponame'
+
+Note: make sure to replace the 'username' (your account username) and the 'reponame' (your image name) accordingly!
+If you use ghcr.io you have to change 'quay.io/username/reponame' with `ghcr.io/NAMESPACE/IMAGE_NAME`
+If you use hub.docker.com you have to change 'quay.io/username/reponame' with `username/image_name`
+
+
+
+[source]
+----
+$ docker login quay.io
+$ docker tag helloworld quay.io/username/helloworld
+----
+
+Check that your image has been tagged:
+
+[source]
+----
+$ docker images
+helloworld                          latest                  6d404b584da2   56 minutes ago   686MB
+quay.io/username/helloworld   latest                  6d404b584da2   56 minutes ago   686MB
+
+----
+Push the image in your registry:
+
+[source]
+----
+$ docker push quay.io/username/helloworld
+Using default tag: latest
+The push refers to repository [quay.io/username/helloworld]
+ad91a69a4efb: Pushed 
+f454069ae0d4: Pushed 
+b21a08aa50a4: Pushed 
+4fc9e93f9cc1: Pushed 
+55ea6d5a354e: Pushed 
+latest: digest: sha256:1d72e7b4f8dd75414a2b1dc8da3505906b0211ac6962e871b84f2c8ead43cb3a size: 1377
+----
+Now your image is available in your container registry (in my case it is available at https://quay.io/repository/rh_ee_msappegr/helloworld)
+
+Note: make sure your registry is public in order to pull it from the cluster. Otherwise you will need to create a https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/[docker-registry secret] .
+
+## Create the YAML value file under the charts folder
+
+In our example the charts folder already exists under quickstart/helloworld/charts, so move in that directory and create a file named values.yaml
```

Comment:
s/in that/to

Reply:
Thank you very much for your review, I substituted `in that` to `to that`. I hope it is what you meant.